docs/configure/images-and-assets.md

```diff
@@ -126,8 +126,6 @@ Upload your files to an online CDN and reference them. In this example, we’re
   csf2Path="configure/images-and-assets#snippet-component-story-static-asset-cdn"
 />
 
-<!-- prettier-ignore-end -->
```

Comment:
Please ensure that the `<CodeSnippets />` are still wrapped with these prettier comments (with a blank line in-between).